### DISCUSSION TITLE
Welcome to Launcher3 Discussions!

### DISCUSSION THREAD

**Discussion by yuchuangu85** [yuchuangu85/Launcher3_mx#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Manny0h2Fresh** [comment]
Hey I'm DunHadditz getting into coding and just exploring and expanding my knowledge
